content/draft/general/keyboard_shortcuts.md

```diff
@@ -0,0 +1,111 @@
+
+#### Message selection shortcuts
+
+requirement: there is already a message in focus
+
+- `E` MUST edit the currently focused message
+    - requirement: the message must have been sent by the current user
+- `Backspace` MUST delete the currently focused message
+    - requirement: the message must have been sent by the current user
+- `P` pins the currently focused message
+    - requirement: the current user must be an admin in a community or a group chat
+- `+` react to currently focused message
+- `R` - reply to currently focused message
+- `CMD + C` - copy currently focused message
+- `OPT + ENTER` - mark currently focused message as read
+
+#### Navigation shortcuts
+
+- `OPT + ENTER + UP` - MUST select the previous community, if there is no previous community then it MUST select the last community
+- `OPT + ENTER + DOWN` - MUST select the next community, if there is no next community then it MUST select the first community
+- `OPT + UP` - MUST select the previous channel, if there is no previous channel then it MUST select the last channel
+- `OPT + DOWN` - MUST select the next channel, if there is no next channel then it MUST select the first channel
+- `CMD + [` - MUST Navigate backward in page history
+- `CMD + ]` - MUST Navigate forward in page history
+- `OPT + SHIFT + UP` - MUST select the previous unread channel, if there is no previous unread channel then it MUST select the last unread channel, if there is no unread channels then it MUST select the first channel
+- `OPT + SHIFT + DOWN` - MUST select the next unread channel, if there is no next unread channel then it MUST select the first unread channel, if there is no unread channel then it MUST select the first channel
+- `OPT + SHIFT + OPT + UP` - MUST select the previous unread channel with mentions, if there is no previous unread channel with mentions then it MUST select the last unread channel with mentions, if there is no unread channels with mentions then it MUST select the first channel
+- `OPT + SHIFT + OPT + DOWN` - MUST select the next unread channel with mentions, if there is no next unread channel with mentions then it MUST select the first unread channel with mentions, if there is no unread channels with mentions then it MUST select the first channel
+
+#### Communities Shortcuts
+requirement: a community must be selected
+
+- `SHIFT + ESCAPE` - MUST mark as community as read
+- `ESCAPE` - MUST mark a community channel as read
+- `CMD + P` - MUST toogle pins popout
+- `CMD + i` - MUST toogle inbox popout
+- `CMD + SHIFT + E` - MUST mark top inbox channel as read
+- `CMD + U` - MUST toggle channel member list
+
+
+#### Chat shortcuts
+
+- `CMD + E` - MUST toggle emoji picker 
+- `CMD + G` - MUST toggle GIF picker 
+- `CMD + S` - MUST toggle sticker picker 
+- `PAGE UP` - MUST Scroll chat up
+- `PAGE DOWN` - MUST Scroll chat down
+- `SHIFT + PAGE UP` - MUST jump to oldest unread message
+- `CMD + SHIFT + H` - MUST trigger Help
```

Comment:
what would this do, i.e what is help here. would it open a particular webpage for example or a certain section in the application?